src/app/core/entity-details/entity-actions-menu/entity-actions-menu.component.ts

```diff
@@ -66,9 +66,9 @@ export class EntityActionsMenuComponent implements OnChanges {
       this.actions = [];
       return;
     }
-    const allActions: EntityAction[] = this.entityActionsMenuService.getActions(
-      this.entity,
-    );
+
+    const allActions: EntityAction[] =
+      this.entityActionsMenuService.getActionsForSingle(this.entity);
 
     // check each action’s `visible` property to hide actions not applicable to the current entity
     const visibleActions = (
```

Comment:
as I put the visible filtering into the service for the `getActionsForBulk` we should probably also move that here into the service method ...

Do you still want to do that, @sadaf895 ?

Reply:
maybe that even means we put the filtering into `getActions()` there, so that it can be used both for single and bulk action methods without duplicating it?